src/reducer/admin/references/possibleValues.js

```diff
@@ -20,17 +28,23 @@ export const getPossibleReferences = (
     reference,
     selectedIds = []
 ) => {
-    const possibleValues = state.admin.references.possibleValues[
-        referenceSource
-    ]
-        ? Array.from(state.admin.references.possibleValues[referenceSource])
-        : [];
+    if (!state.admin.references.possibleValues[referenceSource]) {
+        return null;
+    }
+
+    if (state.admin.references.possibleValues[referenceSource].error) {
+        return state.admin.references.possibleValues[referenceSource];
```

Comment:
why skip the special map / filter processing of the possible values in that case?

Reply:
Because this case corresponds to a fetch error return